photon-api/src/main/scala/com/linkedin/photon/ml/data/RandomEffectDataSetPartitioner.scala

```diff
@@ -93,11 +93,12 @@ object RandomEffectDataSetPartitioner {
   /**
    * Generate a partitioner for one random effect model.
    *
-   * A random effect model is composed of multiple sub-models, each trained on data points for a single item.
-   * We collect the training vector ids that correspond to the random effect, then build an id to partition map.
-   * Data should be distributed across partitions as equally as possible. Since some items have more data points
-   * than others, this partitioner uses simple 'bin packing' for distributing data load across partitions (using
-   * minHeap).
+   * Multiple random effect models, one per random effect ID (e.g. "user123"), are instantiated for a single random
```

Comment:
I know we wrote this one together earlier, but looking at it again, to be consistent with our documentation elsewhere, I think this should change to:

```One random effect model corresponds to one random effect type (e.g. "per-user"). Multiple sub-models, one per ... are instantiated for a single random effect model. Each of these instantiations ...```